scripts/transform-imports.mjs

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env node
+
+/**
+ * Transform Imports Script
+ *
+ * This script helps migrate imports from @commercetools-test-data/* to @commercetools/composable-commerce-test-data/*
+ *
+ * Usage:
+ *   ./scripts/transform-imports.mjs [path]
+ *
+ * Arguments:
+ *   path    Path to the directory containing files to transform (default: current directory)
+ *
+ * The script will:
+ * 1. Find all TypeScript and JavaScript files in the specified directory
+ * 2. Transform imports from @commercetools-test-data/* to @commercetools/composable-commerce-test-data/*
+ * 3. Print the names of files that were updated
+ *
+ * What it transforms ES Module imports
+ *  import { something } from '@commercetools-test-data/core';
+ *  // becomes
+ *  import { something } from '@commercetools/composable-commerce-test-data/core';
+ *
+ * The script ignores files in node_modules, dist, and build directories.
+ */
+
+import { readFileSync, writeFileSync } from 'node:fs';
+import { readdir } from 'node:fs/promises';
+import { join } from 'node:path';
+
+const PACKAGE_JSON = 'package.json';
+const TEST_DATA_SCOPE = '@commercetools-test-data/';
+const NEW_TEST_DATA_PKG = '@commercetools/composable-commerce-test-data';
+const NEW_TEST_DATA_VERSION = '11.0.0';
```

Comment:
We'll need to put the correct version here. I just assume we'll be releasing v11 as the next one.